src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -2327,125 +2328,123 @@ int SketchObject::delConstraintOnPoint(int VertexId, bool onlyCoincident)
     return delConstraintOnPoint(GeoId, PosId, onlyCoincident);
 }
 
-int SketchObject::delConstraintOnPoint(int GeoId, PointPos PosId, bool onlyCoincident)
+// clang-format on

```

Comment:
isn't clang-format on by default?

Reply:
For some files Abdullah had disabled them. See the `clang-format off/on` at the end of the includes and the end of the file.

Unless there's a good reason for that, I plan to remove these once #14696 is completely merged.

Since this code is changing significantly anyway, we can allow clang-format to work here.